open-metadata-implementation/access-services/subject-area/subject-area-server/src/main/java/org/odpi/openmetadata/accessservices/subjectarea/utilities/SubjectAreaUtils.java

```diff
@@ -151,6 +151,19 @@ public static GlossarySummary extractGlossarySummaryFromGlossary(Glossary glossa
         extractNodeSummary(glossary, line, glossarySummary);
         return glossarySummary;
     }
+    /**
+     * Extract category Summary
+     *
+     * @param category the category that is to be summarised
+     * @param line     the line to the category, which feeds part of the node summary
+     * @return Category Summary or null
+     */
+    public static CategorySummary extractGlossarySummaryFromGlossary(Category category, Line line) {
```

Comment:
This method is not used. Please remove it.

Reply:
Removed.